go.mod

```diff
@@ -78,3 +78,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/dapr/kit => github.com/joshvanl/kit v0.0.0-20240923214824-8b9ec753c4bf
```

Comment:
this got merged, so we can use a commit from kit now

Reply:
Thanks :+1: updated